reV/handlers/cli_collect.py

```diff
@@ -257,10 +247,9 @@ def get_node_cmd(name, h5_file, h5_dir, project_points, dsets,
     """
     # make a cli arg string for direct() in this module
     args = ['-f {}'.format(SLURM.s(h5_file)),
-            '-d {}'.format(SLURM.s(h5_dir)),
+            '-cd {}'.format(SLURM.s(collect_pattern)),
```

Comment:
```suggestion
            '-cp {}'.format(SLURM.s(collect_pattern)),
```

Minor typo, but it breaks the whole cli ¯\\ _(ツ) _/¯